pkg/driver/agent.go

```diff
@@ -307,6 +352,16 @@ func (ns *node) NodeUnpublishVolume(
 	klog.Infof("hostpath: volume %s path: %s has been unmounted.",
 		volumeID, targetPath)
 
+	if vol.Spec.SharedMode == apis.LVMExclusiveSharedMode {
+		// Deactivate the LV before sending a successful response
+		// lvchange -an <lv-name>
+		err = lvm.ActivateLVMLogicalVolume(vol, false)
```

Comment:
let us not use name ActivateLVMLogicalVolume when using it for deactivation. We can use function name as ChangeLVMVolume and pass the desired status in the argument.